README.md

```diff
@@ -63,3 +63,26 @@ Dependencies are managed via Composer.
     ```
 
 6. Send us a Pull Request
+
+
+## Examples
+
+SSH into a server with a private key, execute two commands and print the result.
+
+```
+<?php
+
+include 'vendor/autoload.php';
+
+$key = new phpseclib\Crypt\RSA();
+$key->loadKey(file_get_contents('private key'));
```

Comment:
maybe call this `private-key.txt` or something to make it super obvious that this is a file name, because normally file names don't contain spaces